job_templates/ci_job.xml.em

```diff
@@ -337,6 +341,9 @@ if "!CI_COMPILE_WITH_CLANG!" == "true" (
 )
 if "!CI_ENABLE_COVERAGE!" == "true" (
   set "CI_ARGS=!CI_ARGS! --coverage"
+  if "!CI_COVERAGE_FILTER_PKGS!" NEQ "None" (
```

Comment:
The declared default value of the parameter is an empty string, not `None`.

Reply:
d165461e071611edad33e77dcca824f14bbcae6c